Week2/todo_app/program.js

```diff
@@ -21,39 +21,97 @@ class TodoModel {
         });
     }
 
-    create(description, callback) {
+    create(description, userId, callback) {
         // Write code and query to create a new TODO item
+        const createTodoItem = `INSERT INTO todo_items(text, is_completed, user_id) 
+                                VALUES (${dbConnection.escape(description)}, "0", ${dbConnection.escape(userId)})`;
```

Comment:
Did you try whether `"0"` works? Isn't the `is_completed` column has boolean/int data type?